lib/text_client.rb

```diff
@@ -0,0 +1,80 @@
+require 'game'
+require 'text_provider'
+
+class TextClient
+  def initialize
+    @text_provider = TextProvider.new
+  end
+
+  def start
+    game = Game.new()
+    play(game) if game.state == :ready
```

Comment:
is this `if` needed? I'm trying to find a place where a newly initialized game wouldn't be `ready`, but I don't see one

Reply:
You're right.